clang/lib/3C/ConstraintBuilder.cpp

```diff
@@ -68,6 +70,48 @@ void processRecordDecl(RecordDecl *Declaration, ProgramInfo &Info,
   }
 }
 
+void processVarDecl(VarDecl *VD, ProgramInfo &Info,
+                    ASTContext *Context, ConstraintResolver CB) {
+  if(lastRecord2 != nullptr) {
+    auto lastRecordLocation = lastRecord2->getBeginLoc();
+    auto BeginLoc = VD->getBeginLoc();
+    auto EndLoc = VD->getEndLoc();
+    auto VarTy = VD->getType();
+    SourceManager &SM = Context->getSourceManager();
+    bool IsInLineStruct =
+        SM.isPointWithin(lastRecordLocation, BeginLoc, EndLoc)
+        && isPtrOrArrayType(VarTy);
+    bool IsNamedInLineStruct = IsInLineStruct
+                               && lastRecord2->getNameAsString() != "";
+    if (IsNamedInLineStruct) {
+      VDToRDMap2[VD] = lastRecord2;
+      inlines2.insert(VD);
+    }
+    else if (IsInLineStruct && !AllTypes) {
+      CVarOption CV = Info.getVariable(VD, Context);
+      CB.constraintCVarToWild(CV, "Inline struct encountered.");
+    }
+    else if (IsInLineStruct && AllTypes) {
```

Comment:
Group the conditions so that you have the following, equivalent grouping of the above two `else if` clauses:
```
else if (IsInlineStruct) {
  if (!AllTypes) { ... }
  else { ... }
}